### PR TITLE
Update qt doc links

### PR DESCRIPTION
**Pull request by D-Triveni**

### CODE REVIEW

**Review by krish2718 — COMMENTED** [review]
Even the link in `Overview` looks wrong, its point to the same README.md and also the format looks wrong, please fix that as well.

**Review by krish2718 — APPROVED** [review]

**Comment by krish2718** [comment]
> > Even the link in `Overview` looks wrong, its point to the same README.md and also the format looks wrong, please fix that as well.
> 
> It's pointing to the correct pdf only. Can you check in the document page once?

Right, I was trying to see the GH render without waiting for doc, its fine, thanks.

**Comment by D-Triveni** [comment]
> > > Even the link in `Overview` looks wrong, its point to the same README.md and also the format looks wrong, please fix that as well.
> > 
> > 
> > It's pointing to the correct pdf only. Can you check in the document page once?
> 
> Right, I was trying to see the GH render without waiting for doc, its fine, thanks.

ok, Addressing the format change.

File: samples/wifi/wfa_qt_app/README.rst [hunk 1]
```diff
@@ -142,6 +142,9 @@ After programming, the sample shows the following output:
    [00:00:00.835,327] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
    [00:00:00.835,937] <inf> usb_ecm: Set Interface 0 Packet Filter 0x000e not supported
 
+.. note::
+   For running one Security Vulnerability Detection test case (Test case ID: 10003_2), please set Data Packet Type to ICMP in the Test Case Global Configuration section.
```
Comment (richabp) [inline samples/wifi/wfa_qt_app/README.rst:146]:
```suggestion
   For running one Security Vulnerability Detection test case (Test case ID: 10003_2), set **Data Packet Type** to **ICMP** in the **Test Case Global Configuration** section of the QuickTrack Test Tool.
```

File: samples/wifi/wfa_qt_app/README.rst [hunk 2]
```diff
@@ -23,7 +23,7 @@ The QuickTrack sample and library offer support for QuickTrack certification tes
 
 You can choose either of these options for running QuickTrack certification.
 
-See `Wi-Fi Alliance Certification for nRF70 Series <Wi-Fi Alliance Certification for nRF70 Series_>`_ for more information.
+See `Wi-Fi Alliance Certification for nRF70`_ for more information.
```
Comment (richabp) [inline samples/wifi/wfa_qt_app/README.rst:26]:
```suggestion
See `Wi-Fi Alliance Certification for nRF70 Series`_ for more information.
```

**Review by richabp — CHANGES_REQUESTED** [review]

File: samples/wifi/wfa_qt_app/README.rst (hunk 1 above)
Comment (richabp) [reply samples/wifi/wfa_qt_app/README.rst]: Please remove "please".

Comment (D-Triveni) [reply samples/wifi/wfa_qt_app/README.rst]: Removing this complete change as it not needed.

File: samples/wifi/wfa_qt_app/README.rst [hunk 3]
```diff
@@ -108,7 +108,7 @@ The following is an example of the CLI command:
 
 .. code-block:: console
 
-   tunslip6 -s /dev/ttyACM4 -T 2001:db8::1/64
+   ./tunslip6 -s /dev/ttyACM4 -T 2001:db8::1/64
```
Comment (krga2022) [inline samples/wifi/wfa_qt_app/README.rst:111]: Please include the procedure to install tunslip6.

**Review by bama-nordic — APPROVED** [review]

**Review by richabp — APPROVED** [review]

**Comment by cvinayak** [comment]
do we need a tech writer approval here?

**Comment by krish2718** [comment]
> do we need a tech writer approval here?

@richabp  is Wi-Fi techwriter.

**Comment by richabp** [comment]
> do we need a tech writer approval here?

I (TW) have approved it but it seems more changes are in the process.

**Review by richabp — CHANGES_REQUESTED** [review]

**Review by richabp — APPROVED** [review]

**Review by krga2022 — APPROVED** [review]

**Comment by cvinayak** [comment]
Ready to be merged?

**Comment by D-Triveni** [comment]
> Ready to be merged?

Yes, please merge.

**Pull request merged**